docs/self-managed/components/orchestration-cluster/core-settings/concepts/monitoring.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
@buccarel I couldn't find any of these in the Google sheet -- just want to confirm if they need to change for 8.8?